code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -109,6 +109,159 @@
 	name = "Bedhead 3"
 	icon_state = "hair_bedheadv3"
 
+// IPCs
+/datum/sprite_accessory/screen
+	icon = 'icons/mob/ipc_screens.dmi'
+	color_src = null
+
+/datum/sprite_accessory/screen/blank
+	name = "Blank"
+	icon_state = "blank"
+
+/datum/sprite_accessory/screen/pink
+	name = "Pink"
+	icon_state = "pink"
+
+/datum/sprite_accessory/screen/green
+	name = "Green"
+	icon_state = "green"
+
+/datum/sprite_accessory/screen/red
+	name = "Red"
+	icon_state = "red"
+
+/datum/sprite_accessory/screen/blue
+	name = "Blue"
+	icon_state = "blue"
+
+/datum/sprite_accessory/screen/yellow
+	name = "Yellow"
+	icon_state = "yellow"
+
+/datum/sprite_accessory/screen/shower
+	name = "Shower"
+	icon_state = "shower"
+
+/datum/sprite_accessory/screen/nature
+	name = "Nature"
+	icon_state = "nature"
+
+/datum/sprite_accessory/screen/eight
+	name = "Eight"
+	icon_state = "eight"
+
+/datum/sprite_accessory/screen/goggles
+	name = "Goggles"
+	icon_state = "goggles"
+
+/datum/sprite_accessory/screen/heart
+	name = "Heart"
+	icon_state = "heart"
+
+/datum/sprite_accessory/screen/monoeye
+	name = "Mono eye"
+	icon_state = "monoeye"
+
+/datum/sprite_accessory/screen/breakout
+	name = "Breakout"
+	icon_state = "breakout"
+
+/datum/sprite_accessory/screen/purple
+	name = "Purple"
+	icon_state = "purple"
+
+/datum/sprite_accessory/screen/scroll
+	name = "Scroll"
+	icon_state = "scroll"
+
+/datum/sprite_accessory/screen/console
+	name = "Console"
+	icon_state = "console"
+
+/datum/sprite_accessory/screen/rgb
+	name = "RGB"
+	icon_state = "rgb"
+
+/datum/sprite_accessory/screen/golglider
+	name = "Gol Glider"
+	icon_state = "golglider"
+
+/datum/sprite_accessory/screen/rainbow
+	name = "Rainbow"
+	icon_state = "rainbow"
+
+/datum/sprite_accessory/screen/sunburst
+	name = "Sunburst"
+	icon_state = "sunburst"
+
+/datum/sprite_accessory/screen/static

```

Comment:
I don't know if BYOND gets annoyed by this `/static` or not.

Reply:
Better safe than sorry.